packages/drivers/routerlicious-driver/src/tokens.ts

```diff
@@ -21,8 +21,7 @@ export interface ITokenResponse {
 }
 
 /**
- * The ITokenProvider abstracts the token fetching mechanism for a host. Host will be responsible for
- * implementing the interfaces.
+ * Abstracts the token fetching mechanism for a host. Host will be responsible for implementing the interfaces.
```

Comment:
What is the "host" in this context?

Reply:
Let's clarify hosting application

```suggestion
 * Abstracts the token fetching mechanism for a hosting application. The hosting application will be responsible for providing an implementation.
```